### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sulabh007**
acos

### Description

-Adding a new entry:  acos() for PHP 
-Close Issues: #2204

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:2]:
```suggestion
Title: 'acos()'
```

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:3]:
```suggestion
Description: 'Returns the angle whose cosine value is equal to the specified value.'

```

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:17]:
```suggestion
```
Remove extra line break.

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:26]:
```suggestion
Where `number` is the number that the function returns the arc cosine for. Should be specified in range -1 to 1.

```

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:31]: The example must be full PHP code. It needs the `<?php ... ?>` tags and something line an `echo()` function to do something with the result of the function.

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:35]:
```suggestion
The result will be a float value of `0.87629806116834`.
```

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:32]:
```suggestion
```
Remove the comment, as this is handled in the description.

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:36]:
```suggestion
`acos(-1)` returns the value of Pi.

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sulabh007 I did a first review and left a couple of comments. Just a reminder that you shouldn't be making PRs off your `main` branch. You should create a new branch off of each PR. As it is now, you can't submit another PR until this one is merged.

**Comment by sulabh007** [comment]
i have committed those changes

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:32]: You still need to add the `<?php ... ?>` tags to this example.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sulabh007 you missed one comment.

**Comment by sulabh007** [comment]
i have updated the example code

File: content/php/concepts/math-functions/terms/acos/acos.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/acos/acos.md:33]:
```suggestion
  echo(acos(0.64));
```
So we do something with the result of the function.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sulabh007 One more thing.

**Comment by sulabh007** [comment]
changed it

**Comment by sulabh007** [comment]
> @sulabh007 Looks good for a second review.

how does that work

**Comment by SSwiniarski** [comment]
> > @sulabh007 Looks good for a second review.
> 
> how does that work

One of the other maintainers will se the "ready for next review" tag and come and do a second review. We require two maintainer reviews before an entry is merged. Be patient, as there are a lot of PRs for them to go through.

**Review by KTom101 — APPROVED** [review]
@sulabh007 Since my changes were minor, I went ahead and applied them for you in commit [6b7c87a](https://github.com/Codecademy/docs/pull/2289/commits/6b7c87afb1e209b6d441a14e71eabbf2933695e2). 

Note for future entries, it is extremely discouraged to write off your `main` branch. Always make a separate branch when working on a feature. Thanks.

**Comment by KTom101** [comment]
@sulabh007 The entry is live here: https://www.codecademy.com/resources/docs/php/math-functions/acos and you're listed as a contributor: 

Screenshot 2023-06-15 at 9 45 46 PM

Let me know if you notice something amiss.

cc @yangc95

**Pull request merged**
